test/aleph/http_test.clj

```diff
@@ -420,14 +403,14 @@
 
 (deftest test-large-responses
   (with-handler basic-handler
-    (let [pool (http/connection-pool {:connection-options {:response-buffer-size 16}})]
-      (dotimes [i 1 #_1e6]
+    (let [_pool (http/connection-pool {:connection-options {:response-buffer-size 16}})]
```

Comment:
At first I thought to remove the whole `let` form but then I figured that perhaps it's in here as a convenience for tweaking the test somehwo?

Reply:
Base on git blame, this was a partial attempt at deciphering clj-commons/aleph#193. I don't think it works as intended, so maybe the whole thing can be commented-out for now.

Reply:
Thanks for checking! I dug some more and found that this `let` was likely just forgotten to be removed with c031a64cd5110efa5585c7700e8f7de5648ac429. I'll just do that in a separate commit for clarity!